.eslintrc.cjs

```diff
@@ -13,7 +13,7 @@ limitations under the License.
 module.exports = {
   env: {
     browser: true,
-    es2021: true,
+    es2022: true,
```

Comment:
This is required for eslint to support public class fields without requiring babel